RoddyCore/src/de/dkfz/roddy/core/ProjectLoader.groovy

```diff
@@ -236,10 +257,11 @@ class ProjectLoader {
             } else {
                 configurationFile << configurationText
             }
-            return
+        } finally {
+            // Unlock and close the file
+            flock.release()
```

Comment:
* Why do we need this?
* flock.release() cannot fail?
* Is this the flock syscall in Linux? We know it is not working well on NFS4 and in OTP we actually removed all flock-calls to be more portable and become able to use NFS4 and thus get able to use extended attributes and POSIX ACLs. Reconsider.